portal-impl/src/content/Language_fr.properties

```diff
@@ -4985,7 +4985,7 @@ the-selected-list-no-longer-exists=La liste sélectionnée n'existe plus.
 the-selected-role-cannot-be-deleted-because-it-is-a-required-system-role=Le rôle choisi ne peut pas être supprimé car c'est un rôle requis par le système.
 the-selected-web-content-no-longer-exists=Le contenu de Web choisi n'existe plus.
 the-sign-in-process-is-temporarily-disabled-because-the-portal-is-undergoing-a-routine-maintenance-upgrade=Le processus d'authentification est temporairement désactivé parce que le portail est en cours de maintenance. Le processus d'authentification sera réactivé une fois que la mise-à-jour sera complétée. Merci de votre patience.
-the-signature-below-will-be-added-to-each-outgoing-message=La signature ci-dessous sera ajoutée à chaque message envoyé.
+the-signature-below-will-be-added-to-each-outgoing-message=La signature ci-dessous sera ajoutée à chaque message envoyé.
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.